docs/source/getting-started/quickstart.rst

```diff
@@ -97,9 +97,19 @@ Let's apply Laplace noise to a value.
             :end-before: /demo
 
 This is obviously not the easiest way to add noise to a number,
-but it demonstrates a number of OpenDP patterns:
+but it demonstrates a number of low-level OpenDP patterns:
 
-* Defining your metric space with ``space_of`` in Python's Context API, or a (domain, distance) tuple in any language..
-* Chaining operators together with ``>>`` in Python and Rust, or ``|>`` in R.
-* Constructing a ``Measurement`` function on your metric space with ``then_laplace``.
-* Invoking that measurement on a value to get a DP release.
+* First, define your metric space.
+* Then, chain operators together and end with a ``Measurement`` function.
```

Comment:
```suggestion
* Then, chain operators together to construct a ``Measurement`` (aka mechanism).
```

Reply:
Are we still planning to make a bulk change from "measurement" to "mechanism" at some point?

Reply:
Yes, at some point.